src/test/cypress/support/constants.ts

```diff
@@ -17,8 +17,8 @@ export const TIME_FORMAT = 'YYYY-MM-DDTHH:mm:ss.SSS';
 
 // Exercise types
 export enum EXERCISE_TYPE {
-    Text,
-    Modeling,
-    Quiz,
-    Programming,
+    Text = 'text',
```

Comment:
Just curious, why is this change necessary?

Reply:
These were commits included in the old PR, I now rebased it :)